packages/codeeditor/src/editor.ts

```diff
@@ -654,7 +659,8 @@ export namespace CodeEditor {
     tabSize: 4,
     insertSpaces: true,
     matchBrackets: true,
-    autoClosingBrackets: true
+    autoClosingBrackets: true,
+    codeFolding: true
```

Comment:
As with #5750, I think we may want to have this `false` by default (at least to start).